client/components/Rules.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+const Rules = function(props){
+    let children = "This place isn't suitable for children under 12 and the";
+    let rules = {'infants': props.state.state.rules[0].infants, 'pets': props.state.state.rules[0].pets, 'parties': props.state.state.rules[0].events, 'smoking':props.state.state.rules[0].smoking};
```

Comment:
@pauly-van why is this referencing state as a property of itself? maybe be simpler to pass down a specific property so you only have to reference the property name and also easier to identify what the prop is.